src/client/components/card/components/Card.jsx

```diff
@@ -1,14 +1,86 @@
+import React, { PropTypes } from 'react'
```

Comment:
I'd split up the components in here to separate files. Easier for testing and they're large enough to warrant their own file I think.